packages/core/src/js/feedback/utils.ts

```diff
@@ -0,0 +1,17 @@
+export const checkInternetConnection = async (onConnected: () => void, onDisconnected: () => void): Promise<void> => {
+  try {
+    const response = await fetch('https://sentry.io', { method: 'HEAD' });
+    if (response.ok) {
+      onConnected();
+    } else {
+      onDisconnected();
+    }
+  } catch (error) {
+    onDisconnected();
```

Comment:
i: We should add an onError call and pass error as a parameter, so users know they are facing an issue that isn't unrelated to connectivity issues (for example, wrong dsn, server rejecting the feedback, some sdk error,...)

a suggestion for the error message: "Unable to send feedback due to an unexpected error."

Reply:
Good idea 👍 
Added an onError method to communicate the unexpected error with 40b4fd3be9d9f11bee401c972e1efa07aaad1dcf